src/screens/Customers/CustomersEntity.res

```diff
@@ -30,7 +30,15 @@ let getHeading = colType => {
 
 let getCell = (customersData, colType): Table.cell => {
   switch colType {
-  | CustomerId => Text(customersData.customer_id)
+  | CustomerId =>
+    CustomCell(
+      <HSwitchOrderUtils.CopyLinkTableCell
+        url={`/customers/${customersData.customer_id}`}
+        displayValue={customersData.customer_id}
+        copyValue={customersData.customer_id->Some}
```

Comment:
```suggestion
        copyValue={Some(customersData.customer_id)}
```

Can we do in this way as it state better readability

Reply:
we follow arrow notation for props passing, it maintained at all the places

Reply:
I understand but if I say read this Some(x) or x->Some which is better readable. For me I guess in Case of option type I prefer Some(x) so that's why suggested to change